proposals/NNNN-New-vehicle-data-StabilityControlsStatus.md

```diff
@@ -0,0 +1,172 @@
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `GetVehicleData` request: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Boolean" mandatory="false" since="X.x">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `GetVehicleData` response: 
+
+```xml	
+<param name="stabilityControlsStatus" type="StabilityControlsStatus" mandatory="false" since="X.x">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `OnVehicleData` response: 
+
+```xml	
+<param name="stabilityControlsStatus" type="StabilityControlsStatus" mandatory="false" since="X.x">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+### Updates in HMI_API 
+
+#### Add to enum `VehicleDataType` in `Common` interface: 
+
+```xml	
+<element name="VEHICLEDATA_STABILITYCONTROLSSTATUS"/>
+```
+#### Add new struct `StabilityControlsStatus` in `Common` interface:
+
+```xml	
+<struct name="StabilityControlsStatus">
+	<param name="escSystem" type="Common.VehicleDataStatus" mandatory="false">
+		<description>true if vehicle stability control is ON, else false</description>
+	</param>
+	<param name="trailerSwayControl" type="Common.VehicleDataStatus" mandatory="false">
+		<description>true if vehicle trailer sway control is ON, else false</description>
+	</param>
+</struct>
+```
+#### Add to function `SubscribeVehicleData` request: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Boolean" mandatory="false">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `SubscribeVehicleData` response: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Common.VehicleDataResult" mandatory="false">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `UnsubscribeVehicleData` request: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Boolean" mandatory="false">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `UnsubscribeVehicleData` response: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Common.VehicleDataResult" mandatory="false">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `GetVehicleData` request: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Boolean" mandatory="false">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `GetVehicleData` response: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Common.StabilityControlsStatus" mandatory="false">
+	<description>See StabilityControlsStatus</description>
+</param>
+```
+#### Add to function `OnVehicleData` response: 
+
+```xml	
+<param name="stabilityControlsStatus" type="Common.StabilityControlsStatus" mandatory="false">
+	<description>See StabilityControlsStatus</description>
+</param>
```

Comment:
Same suggestion as above, regarding re-organizing for readability.

Reply:
Done